cmd/sonobuoy/app/logs.go

```diff
@@ -31,15 +35,55 @@ func init() {
 		Short: "Dumps the logs of the currently running sonobuoy containers for diagnostics",
 		Run:   getLogs,
 	}
-	// TODO: Determine options
+	cmd.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
 	RootCmd.AddCommand(cmd)
 }
 
 func getLogs(cmd *cobra.Command, args []string) {
-	code := 0
-	if err := ops.GetLogs( /*opts*/ ); err != nil {
+	clientset, err := getClientset()
```

Comment:
we could build a singleton utility function in config that does this.

Reply:
I think I can make LoadClient do a little bit more work and we can use that most places.